### ISSUE TITLE
Snap.parsePathString(...) fails on exponential notation

### ISSUE THREAD

**Issue by mrbeam** [adobe-webplatform/Snap.svg#385]
Using 0.4.1 // build: 2015-04-13

my string:
```
str = "m 476.04409,460.00974 "
	   +"c 36.07942,-36.07942 72.15813,-72.15813 108.23754,-108.23754 "
	   +"c 1.10733,-1.10733 1.10663,-2.90268 0,-4.0093 "
	   +"c-12.02647,-12.02647 -24.05294,-24.05294 -36.07871,-36.07871 "
	   +"c -1.10662,-1.10662 -2.90125,-1.10662 -4.00858,7.1e-4 "
	   +"c -36.07871,36.07871 -72.15813,72.15812 -108.23755,108.23754 "
	   +"c -1.10662,1.10662 -1.10662,2.90126 0,4.00788 "
	   +"c 12.02648,12.02647 24.05295,24.05295 36.07871,36.07871 "
	   +"c 1.10663,1.10662 2.90197,1.10733 4.00859,7.1e-4 "
	   +"z";
```

example: 
https://jsfiddle.net/z9bbpmu0/

thanks for taking care!

**Comment by mrbeam** [comment]
Looks like this regex is incorrect:
pathValues = /(-?\d*\.?\d*(?:e[\-+]?\\d+)?)[\s]*,?[\s]*/gi
String("7.2e-4").match(pathValues)
["7.2", "", "-4", ""]

I hope this helps

**Comment by mrbeam** [comment]
ok got it. Already fixed by this commit: d0085416bbd4fe4b8e86b47c7dcb76833459de28
